transcom/mymove
# B-21489 int Update schema to accommodate multiple transportation offices for office user

## [B-21489](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A1016542)

## Summary

This story is part of a series of backlogs to add the ability for an Office user to have multiple Transportation Offices assigned to them. We do this by adding a new DB table populated with the current Office Users list of Transportation Offices and setting a Primary Office flag to true. 

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Run the DB migration file
2. Open the DB and verify a new transportation_office_assignments table has been created and has all office_users and their appropriate transportation_id with the primary_office flag set to true for all of them.

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
